src/components/UserProfile.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Button,
+  Card,
+  CardContent,
+  CardActions,
+  Table,
+  TableRow,
+  TableCell,
+  getImageListItemBarUtilityClass,
+} from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+// import { getUserProfile } from './api';
+import { Avatar } from '@mui/material';
+import axios from 'axios';
+
+export default function PersonalProfile() {
+  const navigate = useNavigate();
+  const { user, status } = useAuth();
```

Comment:
The `useAuth` hook returns a `user` object that looks like the following:

```json
{
  "user": {
    "_id": "64ef5e90086e1c19b888c72c",
    "name": "Akos",
    "username": "akos",
    "email": "akos123456@example.com",
    "role": "admin",
    "__v": 0,
    "passwordResetExpiresOn": "2023-09-07T17:40:39.477Z",
    "passwordResetToken": "0b38692bf452eb218dd2cc65bf2c8a849bea99e7",
    "id": "64ef5e90086e1c19b888c72c"
  }
}
```

that already has the info we need for the profile and is stored when the user first logs in.

We can store the object in an `authUser` variable to use later in this file:

```suggestion
  const { user: authUser } = useAuth();
```